app/workers/find_and_schedule_for_deletion_suspended_accounts_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class FindAndScheduleForDeletionSuspendedAccountsWorker
```

Comment:
what about shorter name? `DeleteSuspendedAccountsWorker`

Reply:
A shorter name is nice, but that one... it is not really what it does 🤔 It sounds like it actually deletes the accounts...
What about `MarkDeletionSuspendedAccountsWorker`?

Reply:
or `ScheduleDeletionSuspendedAccountsWorker`

Reply:
What about `StaleAccountWorker`?